platform/remarkable/koreader.sh

```diff
@@ -132,7 +132,7 @@ while [ ${RETURN_VALUE} -ne 0 ]; do
         ko_do_fbdepth
     fi
 
-    ./reader.lua >>crash.log 2>&1
+    ./reader.lua "$HOME" >>crash.log 2>&1
```

Comment:
The CI will say to make it ${HOME}
```suggestion
    ./reader.lua "${HOME}" >>crash.log 2>&1
```

Reply:
Actually it would be better to do something like ${args}, like in Kobo. Then you can call `koreader.sh` with arguments that'll be passed to `reader.lua`.

https://github.com/koreader/koreader/blob/6dfe65279c75f37348760fad9f75cb49c8a1eb72/platform/kobo/koreader.sh#L105-L110

Reply:
It makes sense to pass any arguments to koreader.sh through, might be useful